src/normative_only_index.rst

```diff
@@ -13,7 +13,8 @@ It is intended primarily for the developers of software tools which directly con
 
 Users of CellML models may prefer the informative version of the CellML Specification, which is available at :cellml2spec_display:`/`.
 
-A PDF version of the normative specification is available at :cellml2spec_display:`/specification.pdf`.
+The official version of the CellML Normative Specification is available at :cellml2spec_display:`/cellml_2_0_normative_specification.pdf`.
```

Comment:
Is this shown _in_ the official version? I guess that's OK

Reply:
For paper people :)